packages/rocketchat-ui/client/views/app/room.js

```diff
@@ -299,14 +299,6 @@ Template.room.helpers({
 			};
 		});
 		return { buttons };
-	},
```

Comment:
This seems like a bad conflict fix. This feature was added in a previous commit.